src/duration.jl

```diff
@@ -1,62 +1,52 @@
 
 """
-    Duration{T}
+    Duration{T} <: Number
 
 A `Duration` represents a period of time, split into an integer number of seconds and a 
 fractional part.
 
 ### Fields
 - `seconds`: The integer number of seconds.
-- `fraction`: The fractional part of the duration, where `T` is a subtype of `Number`.
+- `fraction`: The fractional part of the duration, where `T` is a subtype of `Real`.
```

Comment:
Should we add this explicitly?

Reply:
Ah this is something I had changed but then reverted back. If you mean restricting the type of the duration parameter to `Real` in the type definition, for me it could be ok! Although I guess it doesn't really matter

Reply:
I would say that we can keep Number and correct the docstring